consensus/core/src/commit.rs

```diff
@@ -76,17 +92,28 @@ pub(crate) struct CommitV1 {
     /// Index of the commit.
     /// First commit after genesis has an index of 1, then every next commit has an index incremented by 1.
     index: CommitIndex,
+    /// Digest of the previous commit.
+    /// Set to CommitDigest::MIN for the first commit after genesis.
+    previous_digest: CommitDigest,
```

Comment:
I believe we could just go with `previous` , the fact that it is a digest is shown by its type

Reply:
Without `digest` it will be less clear at callsites though.